packages/portal/spec-renderer/docs/spec-details.md

```diff
@@ -31,18 +31,18 @@ A Kong UI component for displaying API specs
 Install the component in your host application
 
 ```sh
-yarn add @kong-ui-public/spec-details
+yarn add @kong-ui-public/spec-renderer
 ```
 
 ### Register
 
-You can register `portal-spec-details` globally or locally.
+You can register `spec-details` globally or locally.
```

Comment:
suggestion: We should remove all global registration instructions for this package and the README files; we want all of these components only imported/registered where they are being used.